src/rgw/rgw_sal_motr.cc

```diff
@@ -3681,6 +3684,25 @@ int MotrMultipartUpload::init(const DoutPrefixProvider *dpp, optional_yield y,
     ent.meta.mtime = ceph::real_clock::now();
     ent.meta.user_data.assign(mpbl.c_str(), mpbl.c_str() + mpbl.length());
     ent.encode(bl);
+    std::unique_ptr<RGWObjTags> obj_tags;
+    req_state *s = (req_state *) obj_ctx->get_private();
+    /* handle object tagging */
+    auto tag_str = s->info.env->get("HTTP_X_AMZ_TAGGING");
+    if (tag_str){
+      obj_tags = std::make_unique<RGWObjTags>();
+      int ret = obj_tags->set_from_string(tag_str);
+      if (ret < 0){
+        ldpp_dout(dpp,0) << "setting obj tags failed with rc=" << ret << dendl;
+        if (ret == -ERR_INVALID_TAG){
+          ret = -EINVAL; //s3 returns only -EINVAL for PUT requests
+        }
+        return ret;
+      }
+    }
+    bufferlist tags_1;
```

Comment:
tags_1 what does this name refers to? If nothing, please rename this to some intuitive name. Suggestion, tags_bl or tag_bl

Reply:
Fixed in recent patch